schemas/dex/SCHEMA.md

```diff
@@ -0,0 +1,186 @@
+# DEX Schema
+
+This is OpenBlock Labs standard DEX schema.
+
+## Schema
+
+### Pools
+| Property              | Description                                                          | Type   |
+|-----------------------|----------------------------------------------------------------------|--------|
+| chain_id              | The standard id of the chain.                                        | number |
+| creation_block_number | The block this pool was created in.                                  | number |
+| creation_timestamp    | The timestamp of the block that this pool was created in.            | number |
+| pool_address          | The contract address of the lp pool.                                 | string |
+| lp_token_address      | The token address of the LP token for this pool.                     | string |
+| lp_token_symbol       | The symbol of the LP token.                                          | string |
+| token_address         | The token address of the pool token at token_index.                  | string |
+| token_index           | The index in the pool smart contract that this token appears at (ie, one entry per token in a pool).     | number |
+| fee_rate              | The fee rate of the pool, as a percentage.                           | number |
```

Comment:
for pools with dynamic fee , will this be empty (Optional) ?

if its a 1% pool, will the input here 1 or 0.01?

Reply:
added fee rate to the pool snapshot, so we can have updating fees for dynamic pools